variants/arduino_zero/variant.h

```diff
@@ -94,6 +94,12 @@ extern "C"
 #define PIN_A3               (17ul)
 #define PIN_A4               (18ul)
 #define PIN_A5               (19ul)
+#define PIN_A6               (44ul)
+#define PIN_A7               (45ul)
+#define PIN_A8               (46ul)
+#define PIN_A9               (47ul)
+#define PIN_A10              (48ul)
+#define PIN_A11              (49ul)
 #define PIN_DAC0             (14ul)
```

Comment:
These additional new analog pins will break the logic in: https://github.com/arduino/ArduinoCore-samd/blob/master/cores/arduino/wiring_analog.c#L132-L134

Reply:
this logic is for using analogRead function with 0-5 instead of A0-A5...
so that means:
```
 if (pin < A0) {
    pin += A0;
  }else if(pin > 5 && pin <= 11){
    pin += 38;
 }
```
@sandeepmistry is it ok for you?